### PR TITLE
License Manager implementation

### PR DESCRIPTION
**Pull request by mdurrant-b3**
Provides LicenseManager for v30 and stub for v21

### CODE REVIEW

**Comment by dougwig** [comment]
I usually put a few calls into t.py, then run it against a live ax box, to gather any changes needed in responses.py (there are always a few)

**Comment by mdurrant-b3** [comment]
That's a good idea - I've been writing throwaway smoke testers when I should be adding them to that.

File: acos_client/v21/license_manager.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class LicenseManager(base.BaseV21):
+    """v2.1 LicenseManager is not yet supported"""
+    def create(self, host_list=[], serial=None, instance_name=None,  use_mgmt_port=False,
+               interval=None, bandwidth_base=None, bandwidth_unrestricted=None):
+        raise NotImplementedError("LicenseManager is not yet supported using AXAPI v2.1")
+
+    def get(self):
+        raise NotImplementedError("LicenseManager is not yet supported using AXAPI v2.1")
```
Comment (dougwig) [inline acos_client/v21/license_manager.py]: you can drop the yet on this module, as axapi 2.1 doesn't have a license entry point.  well scratch that, i guess we could add it eventually use cli.deploy

Comment (mdurrant-b3) [inline acos_client/v21/license_manager.py]: Yeah I added it to keep the signatures the same so if someone tried to use it they'd get a clear error as to why it's unusable.  Raising an exception in the body of the class makes `client.py` unhappy as it instantiates the class in the property.

Comment (dougwig) [inline acos_client/v21/license_manager.py]: Oh, I wasn't questioning the diff.  Just whether we could drop the single word 'yet'. But there is a way we could do this, so yet is correct.  Ignore me.

**Comment by mdurrant-b3** [comment]
check

**Comment by dougwig** [comment]
check

**Pull request merged**